### PR TITLE
[IMP]10727 add module add_quota_in_report_menu.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by elicoidal** [comment]
What is Quota? Quant?
Better cancel this one

**Comment by Rona111** [comment]
cc @elicoidal   Quota here means quants.I fix the name and test it on the v8 and add the readme.
@seb-elico @siyuancn @duanyp1991

**Comment by elicoidal** [comment]
name should be stock_quant_report_menu (or similar). Respect Odoo's logic and Always start with the application name

File: add_quants_in_report_menu/README.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
```
Comment (elicoidal) [inline add_quants_in_report_menu/README.rst]: line until the end of the sentence

File: add_quants_in_report_menu/README.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Credits
+=======
+
+Contributors
+------------
+
+* Xia Fajin <xia.fajin@elico-corp.com>
+* Rona Lin <rona.lin@elico-corp.com>
+
+Maintainer
```
Comment (elicoidal) [inline add_quants_in_report_menu/README.rst]: if not OCA, change to Elico Corp

File: add_quants_in_report_menu/README.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Add Quants Menu In Report
+========================
+
+This module extends the functionality of report to add a Quants menu in report for sale man.
```
Comment (elicoidal) [inline add_quants_in_report_menu/README.rst]: This module adds a Quants entry in sales menu and for Accounting & Finance group

File: add_quants_in_report_menu/README.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+* This report set a limit stock value on quant view to Accounting & Finance group .
```
Comment (elicoidal) [inline add_quants_in_report_menu/README.rst]: remove

**Comment by Rona111** [comment]
cc @elicoidal @duanyp1991 @siyuancn @seb-elico

**Comment by elicoidal** [comment]
LGTM

**Comment by Rona111** [comment]
@siyuancn  @seb-elico @duanyp1991 please review.

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**